test/inherited-exports.nix

```
@@ -0,0 +1,31 @@
+{ lib }:
+
+let
+  /* Concatenate a list of strings.
+
+    Type: concatStrings :: [string] -> string
+
+     Example:
+       concatStrings ["foo" "bar"]
+       => "foobar"
+  */
+  concatStrings = builtins.concatStringsSep "";
+
+  /* this should be ignored because it's inherited from an explicit source */
```

If so, then I'd change this to:

```suggestion
  /* this should be ignored because it's shadowed by the `inherit ({}) from;` below */
```